includes/definitions/discovery/stulz.yaml

```diff
@@ -8,177 +8,252 @@ modules:
+                    descr: 'Unit {{ $subindex1 }} Water In Temperature Primary'
+                    index: 'waterInTemperatu1ChillersPrimary.{{ $index }}'
+                    divisor: 10
+                    group: Water In Temperature Primary
+                -
+                    oid: Stulz-WIB8000-MIB::waterInTemperatuhillersSecondary
+                    value: Stulz-WIB8000-MIB::waterInTemperatuhillersSecondary
+                    num_oid: '.1.3.6.1.4.1.29462.10.2.1.1.1.2.1.1.1202.{{ $index }}'
+                    descr: 'Unit {{ $subindex1 }} Water In Temperature Secondary'
+                    index: 'waterInTemperatuhillersSecondary.{{ $index }}'
+                    divisor: 10
+                    group: Water In Temperature Secondary
+                -
+                    oid: Stulz-WIB8000-MIB::waterOutTemperatChillersPrimary
+                    value: Stulz-WIB8000-MIB::waterOutTemperatChillersPrimary
+                    num_oid: '.1.3.6.1.4.1.29462.10.2.1.1.1.2.1.1.1206.{{ $index }}'
+                    descr: 'Unit {{ $subindex1 }} Water Out Temperature Primary'
+                    index: 'waterOutTemperatChillersPrimary.{{ $index }}'
+                    divisor: 10
+                    group: Water Out Temperature Primary
+                -
+                    oid: Stulz-WIB8000-MIB::waterOutTemperatillersSecondary
+                    value: Stulz-WIB8000-MIB::waterOutTemperatillersSecondary
+                    num_oid: '.1.3.6.1.4.1.29462.10.2.1.1.1.2.1.1.1207.{{ $index }}'
+                    descr: 'Unit {{ $subindex1 }} Water Out Temperature Secondary'
+                    index: 'waterOutTemperatillersSecondary.{{ $index }}'
                     divisor: 10
+                    group: Water Out Temperature Secondary
                 -
-                    oid: unitReturnAirTemperature
-                    value: unitReturnAirTemperature
+                    oid: Stulz-WIB8000-MIB::unitAirTemperature
+                    value: Stulz-WIB8000-MIB::unitAirTemperature
+                    num_oid: '.1.3.6.1.4.1.29462.10.2.1.1.1.1.1.1.1.1170.{{ $index }}'
+                    descr: 'Unit {{ $subindex1 }} Air Temperature'
+                    index: 'unitAirTemperature.{{ $index }}'
+                    divisor: 10
+                    group: Air Temperature
+                -
+                    oid: Stulz-WIB8000-MIB::unitReturnAirTemperature
+                    value: Stulz-WIB8000-MIB::unitReturnAirTemperature
                     num_oid: '.1.3.6.1.4.1.29462.10.2.1.1.1.1.1.1.1.1192.{{ $index }}'
+                    descr: 'Unit {{ $subindex1 }} Return Air Temperature'
                     index: 'unitReturnAirTemperature.{{ $index }}'
-                    descr: 'Unit {{ $index }} return temp'
                     divisor: 10
+                    group: Return Air Temperature
                 -
-                    oid: unitWaterInTemperature1
```

Comment:
Make sure not to remove support for other devices.  Discovery will determine which ones are available for polling automatically.